### DISCUSSION TITLE
How to get accurate text width of a text entity with custom font style ?

### DISCUSSION THREAD

**Discussion by harish-simelabs** [mozman/ezdxf#757] [Q&A] [answered]
text_size() didnt return the correct text width.Any other way to get the accurate measurement ?

**Comment by mozman** [comment, marked as answer]
Don’t expect a 100% match compared to CAD applications, `ezdxf` use `Matplotlib` to render text based on TTF-fonts which gives different results than text rendering in AutoCAD. I don't see any way to change this.
